evap/staff/forms.py

```diff
@@ -34,6 +36,8 @@
 from evap.evaluation.tools import clean_email, date_to_datetime
 from evap.results.tools import STATES_WITH_RESULT_TEMPLATE_CACHING, STATES_WITH_RESULTS_CACHING, cache_results
 from evap.results.views import update_template_cache, update_template_cache_of_published_evaluations_in_course
+from evap.staff.importers import ImporterLogEntry
+from evap.staff.importers.base import ImporterDecisionEntry
```

Comment:
I don't like the coupling and all the related problems this has with the importers while not being fully integrated into the process. Essentially, we get the worst of both worlds (code is coupled and UX is cumbersome)

@janno42 Do we want this only for user imports, or should it also work with enrollment imports?